.circleci/config.yml

```diff
@@ -159,13 +160,20 @@ jobs:
           paths:
             - eth-contracts/node_modules
           key: v1-dependencies-{{ checksum "eth-contracts/package.json" }}
-      # run tests!
+      - run:
+          name: run_lint
+          command: |
+            cd eth-contracts
+            npm run lint-circle-ci
+            cd ../
```

Comment:
isn't this already run with `npm run test-circle-ci` below? that runs `scripts/circleci-test.sh` which runs `scripts.lint.sh`